ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -22,6 +22,8 @@ QObject {
     property bool swapProposalReady: false
     property bool swapProposalLoading: false
 
+    property bool showCommunityTokens
```

Comment:
where does this get assigned? we should probably get that store here in that case.

Reply:
it's an optional filter; so it might get assigned from the respective view/component as needed